Proofread UPGRADE guide (#52067)
symfony/symfony: UPGRADE-6.4.md

UPGRADE FROM 6.3 to 6.4
=======================

> Symfony 6.4 and Symfony 7.0 will be released simultaneously at the end of November 2023. According to the Symfony
> release process, both versions will have the same features, but Symfony 6.4 won't include any significant backwards
> compatibility changes.

Reviewer: Using present works for future and past, because most people will read this document after the release.

```suggestion
Symfony 6.4 and Symfony 7.0 are released simultaneously at the end of November 2023. According to the Symfony
release process, both versions have the same features, but Symfony 6.4 doesn't include any significant backwards
compatibility changes.

```
